db.sql

```diff
@@ -37,6 +37,43 @@ CREATE TABLE `conn_bundle` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Dumping data for table `conn_bundle`
+--
+
+LOCK TABLES `conn_bundle` WRITE;
+/*!40000 ALTER TABLE `conn_bundle` DISABLE KEYS */;
+INSERT INTO `conn_bundle` VALUES ('b9967037-4db8-444c-a749-f1f47fc95c93',NULL,0,0,0,14,'test','placeholder','2022-04-21 13:24:36',NULL);
```

Comment:
I don't think we want to have any data in this file.

Reply:
I deleted the data in the db.sql file.  You should see it in the next push.